media/js/devreg/lookup-tool.js

```diff
@@ -51,6 +51,75 @@ require(['prefetchManifest']);
         $(this).next('button').removeClass('disabled');
     });
 
+    $('.add-group button').click(_pd(function() {
+        var button = $(this);
+        var select = button.parent().parent().find('select');
+        var group = select[0].options[select[0].selectedIndex].value;
+        var group_name = select[0].options[select[0].selectedIndex].text;
+        var secret = require('login').userToken();
+
+        $.ajax({
+            url: button.data('api-url') + '?_user=' + encodeURIComponent(secret),
+            data: {
+                group: group
+            },
+            type: button.data('api-method'),
+            dataType: 'text'
+        }).done(function() {
+            notification({
+                message: format(gettext('Group membership "{0}" added'), group_name),
+                timeout: 2000
+            });
+            button.addClass('disabled');
+            select[0].selectedIndex = 0;
+            // add row for newly added group
+            var blankrow = $('.remove-group-blank ');
+            var clone = blankrow.clone(true, true);
+            clone.attr('class', 'remove-group');
+            blankrow.before(clone);
+            clone.find('td:contains("$group_name")')[0].textContent = group_name;
```

Comment:
Since you are using `textContent` here, maybe it'd be better to be consistent and use it above instead of `text` as well.